### ISSUE TITLE
fms2io in input.nml 

### ISSUE THREAD

**Issue by JessicaMeixner-NOAA** [NOAA-EMC/global-workflow#2012]
### What is wrong?

In ufs-weather-model PR: https://github.com/ufs-community/ufs-weather-model/pull/1467 

There was an update to add to the input.nml  the following section 

![Screenshot from 2023-10-31 21-10-44](https://github.com/NOAA-EMC/global-workflow/assets/22549231/becab268-8591-4399-9df3-155247e1a646)

However in PR https://github.com/NOAA-EMC/global-workflow/pull/1933 the addition of the &fms2_io_nml namelist variables were missed.  This should be fixed. 

### What should have happened?

Update ush/parsing_namelists_FV3.sh to add: 
```
&fms2_io_nml
  netcdf_default_format = "netcdf4"
/
```

### What machines are impacted?

All or N/A

### Steps to reproduce

Look at input.nml from rt test and g-w and you will note the missing section. 

### Additional information

None

### Do you have a proposed solution?

Update ush/parsing_namelists_FV3.sh to add: 
```
&fms2_io_nml
  netcdf_default_format = "netcdf4"
/
```

**Comment by WalterKolczynski-NOAA** [comment]
Is there any impact to have been missing this setting?

**Comment by JessicaMeixner-NOAA** [comment]
@DusanJovic-NOAA was the author of the PR where this setting was added, so I believe he'd be a good POC to answer @WalterKolczynski-NOAA question.

**Comment by DusanJovic-NOAA** [comment]
When this parameter (netcdf_default_format) is set to 'netcdf4', FMS will create FV3 restart files in 'netcdf-4' (HDF5) format and chunksizes will be set to horizontal grid dimensions. So it's purely file format (and chunking ) change, the actual values written in restart files should be exactly the same.

**Comment by KateFriedman-NOAA** [comment]
Is this complete now because of #1987? If so, please close, thanks!

**Comment by aerorahul** [comment]
@JessicaMeixner-NOAA Does this issue still need to be addressed?

**Comment by JessicaMeixner-NOAA** [comment]
It was addressed in #1987 as @KateFriedman-NOAA mentioned.
